src/pages/CreateJobProfile/CreateJobProfilePage.jsx

```diff
@@ -0,0 +1,71 @@
+import { Box, Typography } from "@mui/material";
+import TextField from "@mui/material/TextField";
+import Button from "@mui/material/Button";
+import PageLayout from "../../components/PageLayout";
+import { useState } from "react";
+
+/**
+ * Renders the create job profile page.
+ * @returns {JSX.Element} The create job profile page component.
+ */
+function CreateJobProfilePage() {
+
+    const [jobName, setJobName] = useState("");
+    const [jobDescription, setJobDescription] = useState("");
+
+    const handelNext = () => {
+        console.log("Next clicked");
+        console.log("Job Name: " + jobName);
+        console.log("Job Description: " + jobDescription);
+    }
+
+    return (
+        <PageLayout title="Create Job Profile" viewportPage>
+            <Box
+                sx={{
+                    display: "flex",
+                    flexDirection: "column",
+                    justifyContent: "center",
+                    alignItems: "center",
+                    hight: "calc(100% - 60px)",
```

Comment:
```suggestion
                    height: "calc(100% - 60px)",
```
I think this also explains, why the button position is broken